Jenkinsfile

```diff
@@ -87,36 +87,37 @@ pipeline {
                 sh 'mvn -U -B -e clean install -DskipTests -T1C'
             }
         }
-
-        stage('Stable Tests') {
-            steps {
-                echo 'Running tests'
-                // all tests run is very very long (10 hours on Apache Jenkins)
-                sh 'mvn -B -e -fae test '
-                }
-            post {
-                always {
-                    junit(testResults: '**/surefire-reports/*.xml', allowEmptyResults: true)
-                    junit(testResults: '**/failsafe-reports/*.xml', allowEmptyResults: true)
+        stage('Tests') {
+            parallel {
```

Comment:
I would be careful with parallel, that might be why I saw some server already exist exception in some errors in the latest build

Reply:
I'm not sure what you are referring to : 
* [PR 264#1](https://ci-builds.apache.org/blue/organizations/jenkins/james%2FApacheJames/detail/PR-264/1/tests) failures are on OOM and duplicate JMX bindings that I suspect are leftovers bindings of the OOM failures
* [PR 268#51](https://ci-builds.apache.org/blue/organizations/jenkins/james%2FApacheJames/detail/PR-268/51/tests) failures are all related to a DockerCassandraSingleton startup failure which I can't explain at this stage : it occurred in both stable and unstable stages, both of which spawn different JVM and are thus supposed to build their own copy of the image on the first initialization of the singleton. 
* [PR 268#47](https://ci-builds.apache.org/blue/organizations/jenkins/james%2FApacheJames/detail/PR-268/47/tests) failed because of new tests added on master (MailboxChangeTest) I assume I was unlucky to get my PR merged with a new commit which had bad tests, this was fixed after a rebase (I assume the tests were fixed in between :) )
* [PR 268#46](https://ci-builds.apache.org/blue/organizations/jenkins/james%2FApacheJames/detail/PR-268/46/tests) same as previous + a weird cassandra error (I should probably have tagged the CassandraSubscriptionManagerTest as unstable) 
* [PR 268#44](https://ci-builds.apache.org/blue/organizations/jenkins/james%2FApacheJames/detail/PR-268/44/tests) fails on a flaky test
*[PR 268#43](https://ci-builds.apache.org/blue/organizations/jenkins/james%2FApacheJames/detail/PR-268/43/pipeline) failed on a surefire timeout the logs are inconclusive :/

Also of note: 
* using parallel for the unstable tests reduces the build from 3h40 to 2h57
* tests should not bind to fixed ports but maybe there are constraints I'm not aware of ...

Reply:
Thanks for the clarification, I might have raised a false alarm :)